### ISSUE TITLE
May be I didn't get it all...

### ISSUE THREAD

**Issue by NeoScofieldHIT** [revenol/DROO#2]
But I think this is actually supervised learning right? Because you're actually using the output_obj as your ground truth, and the neural network is trying to imitate this ground truth by the mutual entropy loss. In practical situation, output_obj cannot be gained.

**Comment by revenol** [comment]
Note that 'output_obj' is never used to train DROO. DROO works without 'output_obj'.

'output_obj' is loaded only to plot figures. Because different channel conditions generate different rates, it is different to illustrate the converge of the algorithm if we plot the real rate achieved by DROO. Therefore, we introduce a baseline which is the optimal rate saved in 'output_obj', and plot normalized rates = (rate from DROO)/(optimal rate).

Thanks for the comments. More comments are added in the source code to explain this confusion.

**Comment by NeoScofieldHIT** [comment]
> Note that 'output_obj' is never used to train DROO. DROO works without 'output_obj'.
> 
> 'output_obj' is loaded only to plot figures. Because different channel conditions generate different rates, it is different to illustrate the converge of the algorithm if we plot the real rate achieved by DROO. Therefore, we introduce a baseline which is the optimal rate saved in 'output_obj', and plot normalized rates = (rate from DROO)/(optimal rate).
> 
> Thanks for the comments. More comments are added in the source code to explain this confusion.

@revenol Thanks for your kind reply and answer my question. Your open sourced code really helped me a lot. I will continue to follow your works.
